internal/handler/handlers.go

```diff
@@ -253,10 +254,18 @@ func SaveDeveloperProfileHandler(svr server.Server, devRepo devGetSaver, userRep
 			svr.JSON(w, http.StatusBadRequest, "detected_location_id should be set")
 			return
 		}
+		hourlyRate, err := strconv.ParseInt(req.HourlyRate, 10, 64)
+		if err != nil {
+			svr.Log(err, "unable to parse string to int")
+			svr.JSON(w, http.StatusInternalServerError, nil)
+			return
+		}
```

Comment:
might want to add a sanity check on the value hourly rate must be between 0 and 1000